pkg/process/data_test.go

```diff
@@ -18,12 +17,12 @@ type testData struct {
 
 func loadFixture(name string) testData {
 	filename := "./testdata/td" + strings.Title(name) + ".jsonnet"
-	raw, err := ioutil.ReadFile(filename)
-	if err != nil {
-		panic(fmt.Sprint("loading fixture:", err))
-	}
 
-	data, err := jsonnet.Evaluate(filename, string(raw), []string{"./testdata"})
+	vm := jsonnet.MakeVM(jsonnet.Opts{
```

Comment:
Can't use `jsonnet.EvaluateFile` here, because it attempts to resolve the import paths according to the Tanka directory structure, which is not the case here.